src/pages/Result/Result.css

```diff
@@ -0,0 +1,87 @@
+/*main*/
+.result-header {
+  color: var(--primary-color);
+}
+
+.prize-img {
+  height: fit-content;
+  width: 8rem;
+  object-fit: cover;
+}
+
+.result-container {
+  border: 1px solid var(--border-color);
+  border-radius: 1rem;
```

Comment:
It is better to give border radius in px or else as the screen size changes radius also changes and will show unexpected shape.